File: 0004-median-of-two-sorted-arrays/0004-median-of-two-sorted-arrays.cpp
class Solution {
public:
    double findMedianSortedArrays(vector<int>& nums1, vector<int>& nums2) {
        nums1.insert(nums1.end(), nums2.begin(), nums2.end());
        sort(nums1.begin(), nums1.end());
        int a= nums1.size();
        int b= a/2;
        float c;
        if (a==1){
            return nums1[0];
        }
        else if (a%2==1){
            return nums1[b];
        }
        else if (a%2==0){
            c= (nums1[b]+nums1[b-1])/2.0;
            return c;
        }
        return {};
    }
};

File: 0004-median-of-two-sorted-arrays/0004-median-of-two-sorted-arrays.py
class Solution:
    def findMedianSortedArrays(self, nums1: List[int], nums2: List[int]) -> float:
        nums1.extend(nums2)
        nums1.sort()
        n= len(nums1)
        if ((n)%2!=0):
            c= nums1[(int(n/2))]
            return int(c)
        if ((n)%2==0):
            d= nums1[int(((n/2)-0.5))]
            e= nums1[int(((n/2)+0.5))]
            c= (d+e)/2
            return (c)